src/tutorials/touch_basics.md

```diff
@@ -0,0 +1,421 @@
+
+The `draggedIcon` variable is set to the `button` that is pressed. At the same time, the `isHittable` property on the button is disabled. This will allow other controls to be found under the icon.
+
+```lua
+local function OnPressed(button)
+    draggedIcon = button
+    button.isHittable = false
+end
+```
+
+#### Create OnReleased Function
+
+Create a function called `OnReleased`. This function will be called when the [`releasedEvent`](../api/uibutton.md) event has fired when releasing the pointer. For example, if the player lifts their finger it will fire the event.
+
+The `draggedIcon` variable is set to `nil` as the player is no longer dragging an icon around on the screen.
+
+When the event is fired, the function will try to find a control at the `lastPosition`. The `lastPosition` is updated constantly when an icon is dragged around on the screen. Any UI Control that has the property `Is Hittable` enabled can be found using the [`FindControlAtPosition`](../api/ui.md) function. This is useful to see what is below the pointer.
+
+If the `control` matches the `BAG`, then points are awarded, the button is destroyed, and the audio is played. The pitch of the audio is randomized to give it a slightly different sound each time.
+
+If the player releases the pointer, then the [`isHittable`](../api/uibutton.md#properties) property needs to be set to `true` so that it can be picked up and dragged around again.
+
+```lua
+local function OnReleased(button, points)
+    draggedIcon = nil
+
+    local control = UI.FindControlAtPosition(lastPosition)
+
+    if control == BAG then
+        amount = amount + points
+        GEMS_AMOUNT.text = string.format("Gems: %s", amount)
+        button:Destroy()
+
+        AUDIO.pitch = math.random(-100, 600)
+        AUDIO:Play()
+    else
+        button.isHittable = true
+    end
+end
+```
+
+#### Create SpawnIcon Function
+
+Create a function called `SpawnIcon`. This function will spawn a new icon only if nothing is currently being dragged around on the screen. This could be a game mechanic to give the player time to plan. Maybe the icons can be matched when they are added to the bag to increase the score.
+
+An icon is randomly picked from the `ICONS` data table and given some random position and rotation.
+
+The `pressedEvent` and `releasedEvent` are used as they will work on Mobile and PC.
+
+If a [`task`](../api/task.md) has been spawned, then the `repeatInterval` property is set to a random value between 0 and 1 using `math.random`.
+
+```lua
+local function SpawnIcon()
+    if draggedIcon ~= nil then
+        return
+    end
+
+    local icon = World.SpawnAsset(ICON_BUTTON, { parent = CONTAINER })
+    local row = ICONS[math.random(#ICONS)]
+
+    icon:SetImage(row.icon)
+
+    icon.x = math.random(-800, 800)
```

Comment:
is 1600 by 800 just a generic screen size or should work for all screens?

Reply:
Plucked from the air. I came up with those based on the smallest resolution I think. I didn't want to get into keeping things within the bounds of the screen resolution. Easy to add, but want to keep it as basic as possible.